### ISSUE TITLE
peak matrix binarized?

### ISSUE THREAD

**Issue by simon-anders** [calico/scBasset#25]
I'm currently working through your tutorial scripts. I've downloaded your AnnData object of the Buenrostro2018 data, as found [here](https://storage.googleapis.com/scbasset_tutorial_data/buen_ad_sc.h5ad) and trained the model with it. I noticed that the `X` matrix in the AnnData object contains read counts, but if I understand the paper right, you use binarized data. 

Should I hence clip all data to {0,1}, i.e., replace any non-zero count with a one?

I see no mention of this in the tutorial code, but the binary cross entropy loss would not make sense without that, or would it?

Thanks in advance for your help.

**Comment by simon-anders** [comment]
Thanks a lot for the very fast reply. And thanks even more for pointing out the code position. I've been looking through the code for it but missed `generator.call`.
